sympy/stats/tests/test_continuous_rv.py

```diff
@@ -543,6 +543,16 @@ def test_nakagami():
                                 (lowergamma(mu, mu*x**2/omega)/gamma(mu), x > 0),
                                 (0, True))
 
+def test_normal_inverse():
+    a, b = symbols('a b', positive=True)
+    z = Symbol('z', positive=True)
+
+    X = NormalInverse('x', a, b)
+    assert density(X)(z) == sqrt(2)*sqrt(b)*sqrt(z**(-3))*exp(-b*(-a + z)**2/(2*a**2*z))/(2*sqrt(pi))
+    assert E(X) == a
+    assert variance(X) == -a**2 + (a**3 + a**2*b)/b
+    assert cdf(X)(z) == (1/2 - erf(-sqrt(2)*(-sqrt(b)*(1 + z/a)/sqrt(z) - 1)/2)/2)*exp(2*b/a) +\
```

Comment:
Use `S.Half` or `S(1)/2` instead of `1/2` (which is a Python float).

Reply:
Is Rational(1, 2) a good choice? @sidhantnagpal